packages/rollup-dev-tools/test/transpiler/constants-transpilation.spec.ts

```diff
@@ -148,7 +148,7 @@ describe('Solitity contracts should have constants correctly accessible when usi
   const bytesMemoryConstB: Buffer = hexStrToBuf(
     'BBBbeedfeedBBBBBBbeedfeedBBBBBBbeedfeedBBBBBBbeedfeedBBBBBBbeedfeedBBBBBBbeedfeedBBBBBBbeedfeedBBB'
   )
-  it('should work for the first bytes memory constant', async () => {
+  it('should work for the second bytes memory constant', async () => {
```

Comment:
Asked Ben about this, he said that it was just a type-o where he copied a test but forgot to update the name